package.json

```diff
@@ -14,11 +14,12 @@
   "homepage": "http://www.github.com/brave/crypto",
   "main": "index.js",
   "devDependencies": {
-    "browserify": "^16.2.3",
-    "coveralls": "^3.0.0",
+    "browserify": "^16.5.0",
+    "coveralls": "^3.0.9",
     "flow-bin": "^0.39.0",
+    "npm-force-resolutions": "0.0.3",
```

Comment:
After what happened earlier- not sure if this is the best solution. For more details on the error, see https://ci.brave.com/job/ad-block-rust-update/240/console

Basically `brave-core-crx-packager` repo is used for `ad-block-rust-updater` job... and that has a dependency on `https-everywhere-updater` (which had this update). Because of that dependency override, npm was unable to find a proper version of `s3-client`

Reply:
thx, updated this PR to not use that workaround